tests/Core/Tokenizers/Tokenizer/RecurseScopeMapDefaultKeywordConditionsTest.php

```diff
@@ -109,64 +125,158 @@ public static function dataMatchDefault()
      * Note: Cases and default structures within a switch control structure *do* get case/default scope
      * conditions.
      *
-     * @param string   $testMarker    The comment prefacing the target token.
-     * @param int      $openerOffset  The expected offset of the scope opener in relation to the testMarker.
-     * @param int      $closerOffset  The expected offset of the scope closer in relation to the testMarker.
-     * @param int|null $conditionStop The expected offset in relation to the testMarker, at which tokens stop
-     *                                having T_DEFAULT as a scope condition.
-     * @param string   $testContent   The token content to look for.
+     * @param string      $testMarker          The comment prefacing the target token.
+     * @param string|null $testOpenerMarker    The comment prefacing the scope opener token.
+     * @param string|null $testCloserMarker    The comment prefacing the scope closer token.
+     * @param string|null $conditionStopMarker The expected offset in relation to the testMarker, after which tokens stop
+     *                                         having T_DEFAULT as a scope condition.
+     * @param string      $testContent         The token content to look for.
+     * @param bool        $sharedScopeCloser   Whether to skip checking for the `scope_condition` of the
+     *                                         scope closer. Needed when the default and switch
+     *                                         structures share a scope closer. See
+     *                                         https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/810.
      *
      * @dataProvider dataSwitchDefault
      * @covers       PHP_CodeSniffer\Tokenizers\Tokenizer::recurseScopeMap
      *
      * @return void
      */
-    public function testSwitchDefault($testMarker, $openerOffset, $closerOffset, $conditionStop=null, $testContent='default')
-    {
+    public function testSwitchDefault(
+        $testMarker, $testOpenerMarker=null, $testCloserMarker=null, $conditionStopMarker=null, $testContent='default', $sharedScopeCloser=false
+    ) {
```

Comment:
Eh ... no. That's not how a multi-line function signature should look.

Moreover, I don't see a reason for a multi-line signature in the first place. We know we are in the tests, so the parameters don't need to tell us.

```suggestion
    public function testSwitchDefault($testMarker, $openerMarker=null, $closerMarker=null, $conditionStopMarker=null, $testContent='default', $sharedScopeCloser=false) {
```